### PR TITLE
add libwpd

### PR DESCRIPTION
**Pull request by dtardon**

### CODE REVIEW

File: projects/libwpd/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 Google Inc.
```
Comment (Dor1s) [inline projects/libwpd/Dockerfile]: s/2016/2017/ please

File: projects/libwpd/build.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -eu
+# Copyright 2016 Google Inc.
```
Comment (Dor1s) [inline projects/libwpd/build.sh]: 2017

File: projects/libwpd/build.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+pushd librevenge
+./autogen.sh
+./configure --without-docs --disable-shared --enable-static --disable-tests
+make -j$(nproc)
+rvnginc=$(pwd)/inc
+rvnglib=$(pwd)/src/lib
+popd
+
+./autogen.sh
```
Comment (Dor1s) [inline projects/libwpd/build.sh]: I see that you append `-std=c++11` to `$CXX` in a separate librevenge build (https://github.com/google/oss-fuzz/blob/master/projects/librevenge/build.sh#L18), don't you need it here as well?

Comment (dtardon) [inline projects/libwpd/build.sh]: It is not needed--I guess I should remove the line from librevenge (and from everywhere else I used it).

**Comment by dtardon** [comment]
c.f. https://github.com/google/oss-fuzz/pull/499#issuecomment-291251656

**Pull request closed without merging**